hammer/reporting-remediation/reporting/create_iam_key_rotation_tickets.py

```diff
@@ -105,6 +106,20 @@ def create_jira_ticket(self):
 
                     IssueOperations.set_status_reported(ddb_table, issue)
 
+                    comment = f"Stale access key '{key_id}' issue is going to be remediated in " \
+                              f"{retention_period} days"
+                    slack.report_issue(
```

Comment:
What if remediation is off? It seems the message will be posted to slack/jira that the issue is going to be remediated in 365 days, https://github.com/dowjones/hammer/blob/dev/hammer/library/config.py#L547

Reply:
Added conditional post to slack/jira based on remediation flag.